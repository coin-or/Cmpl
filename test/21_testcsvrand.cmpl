const:
    
	a[,] := readcsv("testcsvmatrix.csv"); 	
	echo "read matrix done"; 
        c[] := readcsv("testcsvvector.csv");  
	echo "read vector done";  
 	
var:
    x[1..dim(c[])]: real[0..];
    
objective:
   c[]T * x[] ->max;
    
constraint:
   a[,] * x[] <= 100;
   echo "x "+  dim(x[]);
   echo "a "+ dim(a[1,]) + " " +dim(a[,1]);
     


