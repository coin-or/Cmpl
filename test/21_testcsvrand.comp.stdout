read matrix done
read vector done
x 200
a 200 200
